src/App.js

```diff
@@ -1,12 +1,27 @@
 import React, { Component } from 'react';
 import './App.css';
 import Header from './Header';
+import Menu from './Menu';
 
 class App extends Component {
+  constructor(props) {
+  	super(props);
+  	this.state = {
+  	  menuVisible: false,
```

Comment:
Small thing, but I like to preface boolean states with `is` -- i.e. `isMenuVisible`